src/tools/vision/net.cpp

```diff
@@ -63,6 +63,7 @@ class FireVisionNetworkTool
   /** Constructor.
    * @param argp argument parser
    */
+	explicit
```

Comment:
The `explicit` keyword is usually on the same line as the c'tor definition.

Reply:
Agreed.